### PR TITLE
AGT-6 backend dummy test

### PR DESCRIPTION
**Pull request by Jerry-ZWL**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea
 /node_modules
-/dist/app
\ No newline at end of file
+/dist/app
+/package-lock.json
```
Comment (beltekylevi) [inline .gitignore]: `package-lock.json` is a new npm file, it is required. if you want to know more about it:
https://docs.npmjs.com/files/package-lock.json

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # epam-jsa-agate
 EPAM-JSA-AGATE --Local Business Finder (Yelp)
+
```
Comment (beltekylevi) [inline README.md:3]: Remove unnecessary empty line

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
...
+
+#run mocha test in command line:
+./node_modules/mocha/bin/mocha $(find test/mocha -name '*js')
```
Comment (beltekylevi) [inline README.md]: i think you can run this by `npm run test:backend` it's more developer friendly

File: package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "description": "EPAM-JSA-AGATE --Local Business Finder (Yelp)",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha $(find test/mocha -name '*js')"
```
Comment (beltekylevi) [inline package.json]: task name should be `test:backend` also just tell mocha which folder is where your backend tests are

File: test/mocha/test.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+var assert = require('assert');
```
Comment (beltekylevi) [inline test/mocha/test.js]: it can be const

File: test/mocha/test.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+describe('Mocha Test', function () {
```
Comment (beltekylevi) [inline test/mocha/test.js]: rename it to like Dummy Backend Test or something

File: test/mocha/test.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+  it('dummy test', function () {
```
Comment (beltekylevi) [inline test/mocha/test.js]: And this maybe to true is true

File: test/mocha/test.js [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+    assert.equal(true, true);
```
Comment (beltekylevi) [inline test/mocha/test.js]: you might use assertTrue(true), but it doesn't really matter, its more like  a comment than a change request.

**Review by beltekylevi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Jerry-ZWL) [reply README.md:3]: blank lines removed

File: test/mocha/test.js (hunk 5 above)
Comment (Jerry-ZWL) [reply test/mocha/test.js]: yep changed to const

File: test/mocha/test.js (hunk 6 above)
Comment (Jerry-ZWL) [reply test/mocha/test.js]: Dummy Backend Test as required

File: test/mocha/test.js (hunk 7 above)
Comment (Jerry-ZWL) [reply test/mocha/test.js]: make sense

File: test/mocha/test.js (hunk 8 above)
Comment (Jerry-ZWL) [reply test/mocha/test.js]: checked node.js doc/
no assertTrue
but assert.ok does same work

File: package.json [hunk 9]
```diff
@@ -4,7 +4,7 @@
   "description": "EPAM-JSA-AGATE --Local Business Finder (Yelp)",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test:backend": "mocha $(find test/mocha -name '*js')"
```
Comment (beltekylevi) [inline package.json]: just use the simple folder version

File: test/mocha/test.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+const assert = require('assert');
+describe('Mocha Test', function () {
```
Comment (beltekylevi) [inline test/mocha/test.js]: `Dummy Backend Test` here

File: test/mocha/test.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
...
+  it('Dummy Backend Test', function () {
```
Comment (beltekylevi) [inline test/mocha/test.js]: `True is true` here

**Review by beltekylevi — CHANGES_REQUESTED** [review]

File: test/mocha/test.js [hunk 12]
```diff
@@ -0,0 +1,6 @@
+const assert = require('assert');
+describe('Dummy Backend Test', function () {
```
Comment (beltekylevi) [inline test/mocha/test.js]: add an empty line after the imports

**Review by beltekylevi — CHANGES_REQUESTED** [review]

File: README.md [hunk 13]
```diff
@@ -1,2 +1,5 @@
 # epam-jsa-agate
 EPAM-JSA-AGATE --Local Business Finder (Yelp)
+
+#run mocha test in command line:
```
Comment (beltekylevi) [inline README.md]: add a space after #

**Review by beltekylevi — CHANGES_REQUESTED** [review]

**Review by beltekylevi — APPROVED** [review]

**Pull request merged**